source/configuration_manual/config_file/config_variables.rst

```diff
@@ -54,6 +53,17 @@ If :ref:`var_expand_crypt_plugin` is loaded, these also work globally:
 |                               | .. dovecotadded:: 2.2.29    |
 +-------------------------------+-----------------------------+
 
+.. _variables-system-variables:
+
+Supported system variables
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+``cpu_count``
+  Number of CPUs available. Works only on Linux and FreeBSD-like systems.
+``hostname``
+  Hostname (without domain). Can be overridden with DOVECOT_HOSTNAME
+  environment variable.
```

Comment:
\`\`DOVECOT_HOSTNAME\`\` and could also mention it's needed to be set in :ref:import_environment setting.